pkgs/development/python-modules/openfga-sdk/default.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonAtLeast,
+  pytestCheckHook,
+  aiohttp,
+  build,
+  mock,
+  opentelemetry-api,
+  pytest-asyncio,
+  pytest-cov-stub,
+  python-dateutil,
+  setuptools,
+  urllib3,
+}:
+
+buildPythonPackage rec {
+  pname = "openfga-sdk";
+  version = "0.9.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "openfga";
+    repo = "python-sdk";
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
`rev` should only be used for commits